src/ts/components/ads/AdMessageLabel.ts

```diff
@@ -1,51 +1,50 @@
-import {Label, LabelConfig} from '../labels/Label';
 import {UIInstanceManager} from '../../UIManager';
-import {StringUtils} from '../../utils/StringUtils';
 import { AdEvent, LinearAd, PlayerAPI } from 'bitmovin-player';
-import { i18n } from '../../localization/i18n';
+import { Label, LabelConfig } from '../labels/Label';
 
 /**
- * A label that displays a message about a running ad, optionally with a countdown.
+ * A label that displays a message regarding the ad that's currently being played.
  *
- * @category Components
+ * @category Labels
  */
 export class AdMessageLabel extends Label<LabelConfig> {
+  private adMessage: string = '';
 
   constructor(config: LabelConfig = {}) {
     super(config);
 
     this.config = this.mergeConfig(config, {
-      cssClass: 'ui-label-ad-message',
-      text: i18n.getLocalizer('ads.remainingTime') ,
+      cssClasses: ['label-metadata', 'label-metadata-title'],
     }, this.config);
   }
 
   configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
     super.configure(player, uimanager);
 
-    let config = this.getConfig();
-    let text = config.text;
-
-    let updateMessageHandler = () => {
-      this.setText(StringUtils.replaceAdMessagePlaceholders(i18n.performLocalization(text), null, player));
+    const init = () => {
+      this.setText(this.adMessage);
     };
 
-    let adStartHandler = (event: AdEvent) => {
-      let uiConfig = (event.ad as LinearAd).uiConfig;
-      text = uiConfig && uiConfig.message || config.text;
+    const unload = () => {
```

Comment:
Since this is not only called from unload but also from other events please name it in a way describing what it does. 

```suggestion
    const clearAdMessageText = () => {
```

Reply:
Address in dccaf9354c3e125aeecc955ae629f1b3e39abd22